fastlane/Fastfile

```diff
@@ -300,12 +358,13 @@ def is_snapshot_version?(version_name)
 end
 
 def replace_in(previous_text, new_text, path, allow_empty=false)
+  # temporary until we have shared plugin in place
   if new_text.to_s.strip.empty? and not allow_empty
     UI.user_error!("Missing `new_text` in call to `replace_in`, looking for replacement for #{previous_text} 😵.")
   end
-  sed_regex = 's|' + previous_text.sub(".", "\\.") + '|' + new_text + '|'
-  backup_extension = '.bck'
-  sh("sed", '-i', backup_extension, sed_regex, path)
+  original_text = File.read(path)
+  replaced_text = original_text.gsub(previous_text, new_text)
+  File.write(path, replaced_text)
```

Comment:
I took this from the plugin https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal, which will be added in https://github.com/RevenueCat/purchases-hybrid-common/pull/186. The `sed` command wasn't working in the docker machine.